facdb/sql/foodbankny_foodbanks.sql

```diff
@@ -0,0 +1,38 @@
+DROP TABLE IF EXISTS _foodbankny_foodbanks;
+
+SELECT
+    uid,
+    source,
+    name as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    address,
+    city,
+    zip_code as zipcode,
+    NULL as boro,
+    NULL as borocode,
+    NULL as bin,
+    NULL as bbl,
+    (CASE
+		WHEN program_type ~* 'pantry' THEN 'Food Pantry'
+		WHEN program_type ~* 'Soup Kitchen' THEN 'Soup Kitchen'
+	END) as factype,
+    (CASE
+	    WHEN program_type ~* 'senior' THEN 'Senior Services'
```

Comment:
Since we're excluding records where  program_type !~* 'senior' should this just be 
`Soup Kitchens and Food Pantries' as facsubgrp`?